snap/hooks/install

```diff
@@ -2,58 +2,33 @@
 
 set -xe
 
-# Create some directories for bind
-mkdir -p "$SNAP_DATA/etc"
-mkdir -p "$SNAP_DATA/usr/share"
-mkdir -p "$SNAP_DATA/var/cache/bind"
+$SNAP/bin/prepare
 
-# Copy the config files to SNAP_DATA
-cp -rf --preserve=mode "$SNAP/etc/bind" "$SNAP_DATA/etc/bind"
-cp -rf --preserve=mode "$SNAP/usr/share/dns" "$SNAP_DATA/usr/share"
+# Create some directories
+mkdir -p $SNAP_COMMON/api
+
+# Copy the bind config files to SNAP_COMMON
+cp -rf --preserve=mode "$SNAP/etc/bind" "$SNAP_COMMON/bind"
 
 # Fix the config files to take the new location into account
 SNAP_CURRENT="/var/snap/$SNAP_NAME/current"
-for file in $SNAP_DATA/etc/bind/* ; do
+for file in $SNAP_COMMON/bind/* ; do
```

Comment:
Same here, quotes

Reply:
This is coming in a subsequent PR: https://github.com/canonical/charmed-bind/pull/23

Reply:
Ok. But it's simpler to review it when added/changed in the same PR so avoid some instances of this slipping unnoticed, though

Reply:
Makes sense